openapi.json

```diff
@@ -3,7 +3,7 @@
     "info": {
         "title": "groupfolders",
         "version": "0.0.1",
-        "description": "Admin configured folders shared with everyone in a group",
+        "description": "Admin configured folders shared with everyone in a Team",
```

Comment:
These changes are ambiguous, because circles are called Teams now, so this make it sound like it’s specific to teams/circles, no?

Reply:
We have the same situation in other places as well.
I think calling it `group/Team` or `group or Team` makes it harder to understand.
So I would try to use `Team` everywhere, except where we really mean `group` on a technical level.

Reply:
I asked @marcoambrosini and he seems to agree, so let's keep it like it is now.